src/Compilers/CSharp/Portable/Emitter/EditAndContinue/PEDeltaAssemblyBuilder.cs

```diff
@@ -304,6 +304,14 @@ public override IEnumerable<Cci.INamespaceTypeDefinition> GetTopLevelSourceTypeD
             return _previousDefinitions.TryCreateVariableSlotAllocator(_previousGeneration, Compilation, method, topLevelMethod, diagnostics);
         }
 
+        internal override MethodInstrumentation GetMethodBodyInstrumentations(MethodSymbol method)
+        {
+            // EmitDifference does not allow settings these:
```

Comment:
>settings these [](http://example.com/codeflow?start=45&length=14)

"setting these"?

Reply:
Changed to "... settings instrumentation kinds on EmitOptions"